source/blog/2024-10-14-doctrine-core-team-meetup-2024.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Doctrine Core Team Meetup, ORM 2.19.8, 2.20.0, 3.2.3, 3.3.0 Releases, DBAL 4.2.0"
```

Comment:
I don't think it's worth mentioning 3.2.3, it contains nothing interesting for the end user. My mistake.